rocAL/rocAL/source/meta_data/meta_node_flip.cpp

```diff
@@ -42,16 +42,16 @@ void FlipMetaNode::update_parameters(MetaDataBatch* input_meta_data)
     vxCopyArrayRange((vx_array)_flip_axis, 0, _batch_size, sizeof(int),_flip_axis_val.data(), VX_READ_ONLY, VX_MEMORY_TYPE_HOST);
     for(int i = 0; i < _batch_size; i++)
     {
-        auto bb_count = input_meta_data->get_bb_labels_batch()[i].size();
-        BoundingBoxLabels labels_buf;
+        auto bb_count = input_meta_data->get_label_batch()[i].size();
         BoundingBoxCords coords_buf;
+        Labels labels_buf;
         coords_buf.resize(bb_count);
         labels_buf.resize(bb_count);
         memcpy(labels_buf.data(), input_meta_data->get_bb_labels_batch()[i].data(),  sizeof(int)*bb_count);
         memcpy((void *)coords_buf.data(), input_meta_data->get_bb_cords_batch()[i].data(), input_meta_data->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
         BoundingBoxCords bb_coords;
-        
-        for(uint j = 0; j < bb_count; j++)
+        Labels bb_labels;
```

Comment:
Why is this variable created and set here?
But the values are not updated in this variable